### DISCUSSION TITLE
Release cycle?

### DISCUSSION THREAD

**Discussion by JanKru** [gethyas/hyas#428] [Q&A] [answered]
Hi Guys

**DISCLAIMER**: I've only been using Hyas for a few days, so my question might be stupid.

I ask when @hyas/seo will get the next release? My main concern is the following file `@hyas/seo/layouts/partials/hugo-seo/private/params` line 43

``` 
{ $locale_alternate = slice }} 
``` 

which should be
```
which should be {{ $locale_alternate := slice }}.
```

This leads, for example, to multilingual no longer working at [Doks](https://github.com/h-enk/doks)

For example settings in `hugo.toml`

```
# Multilingual
defaultContentLanguage = "en"
disableLanguages = ["nl"]
```

will cause 

```
...error calling partial: "/Users/jan/Projects/j1n-docs-blog/node_modules/@hyas/seo/layouts/partials/hugo-seo/private/params/og.html:43:25": execute of template failed: template: partials/hugo-seo/private/params/og.html:43:25: executing "partials/hugo-seo/private/params/og.html" at <slice>: undefined variable: $locale_alternate
```

**Comment by h-enk** [comment, marked as answer]
Thanks for the nudge. There you go: https://github.com/gethyas/hyas#packages

**Reply by JanKru** [reply]
Thx for the quick reply & fix !
